File: app/src/main/java/com/gtn/abhis/crackit4/HowToPlay.java
package com.gtn.abhis.crackit4;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class HowToPlay extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_how_to_play);
    }

    @Override
    public void onBackPressed() {

        finish();
        Intent intent = new Intent(HowToPlay.this, Login_Page.class);
        intent.setFlags(Intent.FLAG_ACTIVITY_NEW_TASK | Intent.FLAG_ACTIVITY_CLEAR_TASK);
        startActivity(intent);
    }
}

File: app/src/main/java/com/gtn/abhis/crackit4/HowToPlay.kt
package com.gtn.abhis.crackit4

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class HowToPlay : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_how_to_play)
    }

    override fun onBackPressed() {
        finish()
        val intent = Intent(this@HowToPlay, LoginPage::class.java)
        intent.flags = Intent.FLAG_ACTIVITY_NEW_TASK or Intent.FLAG_ACTIVITY_CLEAR_TASK
        startActivity(intent)
    }
}